backend/discord_client.py

```
@@ -0,0 +1,329 @@
+                mention_author=mention_author,
+                view=view,
+                poll=poll,
+            )
+
+        try:
+            message = await self.rate_limiter.execute_with_retry(
+                _send,
+                endpoint,
+            )
+            logger.debug(f"Message sent to channel {channel}")
+            return message
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)